### ISSUE TITLE
v4.2 setting icon color

### ISSUE THREAD

**Issue by acoreyj** [ionic-team/ionicons#568]
With version 4.2 nothing seems to change the color of the icons. I have tried setting the color property in various places but it says black. Setting color on the parent property works fine on 4.0 and 4.1

**Comment by manucorporat** [comment]
@acoreyj what CSS are you using to change the color?

**Comment by ralexrdz** [comment]
same behavior for me

**Comment by acoreyj** [comment]
Essentially this works with 4.0 and 4.1

```html
<ul style="color:white">
<li>
	<ion-icon name="create"></ion-icon>
</li>
</ul>
```
I tried putting the color style on many different spots though with no luck.
I should mention I'm using the unpkg script.

How are we meant to change the color?

**Comment by manucorporat** [comment]
<img width="427" alt="screen shot 2018-06-26 at 21 21 59" src="https://user-images.githubusercontent.com/127379/41934452-08decf36-7987-11e8-9598-a6a9636943bb.png">
it works to me...

**Comment by manucorporat** [comment]
Closing since this was fixed in `4.2.1`

**Comment by barocsi** [comment]
https://github.com/ionic-team/ionic/blob/master/angular/BREAKING.md#icon
it says we should use fill instead of color. But fill doesnt work, color works.
Ideas?
